domains/ahsan.json

```diff
@@ -5,7 +5,7 @@
     "email": "ahsan.qureshi8@gmail.com"
   },
   "record": {
-    "CNAME": "www.thisisahsan.com",
+    "CNAME": "https://www.thisisahsan.com",
```

Comment:
This is an invalid CNAME record. If you are having trouble with your domain, you can close this pr and create a support issue and I'll try to help you as soon as I can.